data/persons.txt

```diff
@@ -1850,3 +1850,187 @@ ship "Modified Ladybug"
 	explode "huge explosion" 10
 	"final explode" "final explosion large"
 	description `This is an old war-torn Hai Ladybug with numerous modifications done on the hull. Apart from the additional turret mounts, the main adjustments on the vessel are three huge engine cells - two in the back replacing the old engine pods, and one in the front replacing the gun mounts. These push the Modified Ladybug to speeds only an insane captain would enjoy.`
+
+
+
+person "Patrol Team"
+	government "Author"
+	frequency 300
```

Comment:
```suggestion
	frequency 300
```
I want to keep an eye on this. Generally we want to the original person ships to be more common, so if it appears that this fleet is appearing more than necessary, this might change later (I had to do so for Gefu's pers ship before). Just leaving this as a note.